WordPress/src/main/java/org/wordpress/android/ui/stories/StoryComposerViewModel.kt

```diff
@@ -0,0 +1,72 @@
+package org.wordpress.android.ui.stories
+
+import android.content.Intent
+import androidx.lifecycle.ViewModel
+import org.wordpress.android.fluxc.model.LocalOrRemoteId.LocalId
+import org.wordpress.android.fluxc.model.PostImmutableModel
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.push.NotificationType
+import org.wordpress.android.ui.notifications.SystemNotificationsTracker
+import org.wordpress.android.ui.posts.EditPostRepository
+import org.wordpress.android.ui.posts.PostEditorAnalyticsSession
+import org.wordpress.android.util.analytics.AnalyticsUtilsWrapper
+import javax.inject.Inject
+
+class StoryComposerViewModel @Inject constructor(
+    private val systemNotificationsTracker: SystemNotificationsTracker,
+    private val saveInitialPostUseCase: SaveInitialPostUseCase,
+    private val analyticsUtilsWrapper: AnalyticsUtilsWrapper
+) :
+        ViewModel() {
+    private lateinit var editPostRepository: EditPostRepository
+    private lateinit var site: SiteModel
+    private lateinit var postEditorAnalyticsSession: PostEditorAnalyticsSession
+
+    fun start(
+        editPostRepository: EditPostRepository,
+        postId: LocalId,
```

Comment:
For the `postId` I am using the `LocalId` wrapper class so that it's easier to identify the id as being a local one versus remote.